pyEpiabm/pyEpiabm/tests/test_sweep/test_update_place_sweep.py

```diff
@@ -1,54 +1,93 @@
 import unittest
+from unittest import mock
 
-import pyEpiabm as pe
+from pyEpiabm.property.infection_status import InfectionStatus
+from pyEpiabm.property.place_type import PlaceType
+from pyEpiabm.core.population import Population
+from pyEpiabm.core.parameters import Parameters
+from pyEpiabm.sweep.update_place_sweep import UpdatePlaceSweep
 
 
 class TestUpdatePlaceSweep(unittest.TestCase):
     """Test the 'UpdatePlaceSweep' class.
     """
 
-    @classmethod
-    def setUpClass(cls) -> None:
+    def setUp(self) -> None:
         """Initialises a population with one infected person. Sets up a
         single household containing this person.
         """
-        cls.pop = pe.Population()
-        cls.pop.add_cells(1)
-        cls.cell = cls.pop.cells[0]
-        cls.pop.cells[0].add_microcells(1)
-        cls.microcell = cls.cell.microcells[0]
-        cls.pop.cells[0].microcells[0].add_people(1)
-        cls.person = cls.pop.cells[0].microcells[0].persons[0]
-        cls.person.infection_status = pe.property.InfectionStatus.InfectMild
-        cls.microcell.add_place(1, (1, 1), pe.property.PlaceType.Hotel)
-        cls.place = cls.cell.places[0]
-        pe.Parameters.instance().time_steps_per_day = 1
-        cls.time = 1
+        self.pop = Population()
+        self.pop.add_cells(1)
+        self.cell = self.pop.cells[0]
+        self.pop.cells[0].add_microcells(1)
+        self.microcell = self.cell.microcells[0]
+        self.pop.cells[0].microcells[0].add_people(1)
+        self.person = self.pop.cells[0].microcells[0].persons[0]
+        self.person.update_status(InfectionStatus.InfectMild)
```

Comment:
Do you need to artificially make this person infectious too? (or infact do they need to be infectious at all?)

Reply:
apparently not